File: src/main/java/com/example/backend/dto/system/JobDto.java
package com.example.backend.dto.system;

import com.example.backend.documents.JobStatus;
import com.example.backend.documents.JobType;

import java.time.Instant;
import java.util.List;
import java.util.UUID;

public record JobDto(
    UUID id,
    Instant createDate,
    Instant updateDate,
    JobType type,
    JobStatus status,
    List<JobResultDto> results
) {
}

File: src/main/kotlin/com/example/backend/dto/system/JobDto.kt
package com.example.backend.dto.system

import com.example.backend.documents.JobStatus
import com.example.backend.documents.JobType
import java.time.Instant
import java.util.*

data class JobDto(
    var id: UUID?,
    var createDate: Instant?,
    var updateDate: Instant?,
    var type: JobType?,
    var status: JobStatus?,
    var results: List<JobResultDto>?
)


